aidants_connect_web/tests/test_views.py

```diff
@@ -165,7 +165,7 @@ def test_post_to_recap_without_sub_creates_error(self):
         messages = list(get_messages(response.wsgi_request))
         self.assertEqual(len(messages), 1)
 
-    def test_post_to_recap_checks_existing_sub_and_updates_info(self):
+    def test_post_to_recap_checks_existing_sub_and_doesnt_update_info(self):
```

Comment:
Should this test pass ?
Here is my reasoning:
The `sub` is supposed to guarantee that the `user_info` will be the same.
If it were to send us different information, we should throw an error, even though the risk of this happening is very low.

I think that a more relevant test would be to create a user with a `creation_date=x` and test that the returned user has the same creation date proving that another instance of the same user has not been created.